google/resource_compute_instance_group_migrate_test.go

```diff
@@ -8,24 +8,43 @@ import (
 
 func TestComputeInstanceGroupMigrateState(t *testing.T) {
 	cases := map[string]struct {
-		StateVersion int
-		Attributes   map[string]string
-		Expected     map[string]string
-		Meta         interface{}
+		StateVersion       int
+		Attributes         map[string]string
+		ExpectedAttributes map[string]string
+		ExpectedId         string
+		Meta               interface{}
 	}{
-		"change instances from list to set": {
+		"v1 to v2": {
```

Comment:
wait just kidding, I forgot to actually pull your latest commit. Nothing to see here.